spec/tests/ladybug_energy_schedule_spec.rb

```diff
@@ -0,0 +1,77 @@
+# and/or other materials provided with the distribution.
+#
+# (3) Neither the name of the copyright holder nor the names of any contributors
+# may be used to endorse or promote products derived from this software without
+# specific prior written permission from the respective party.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDER(S) AND ANY CONTRIBUTORS
+# "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO,
+# THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER(S), ANY CONTRIBUTORS, THE
+# UNITED STATES GOVERNMENT, OR THE UNITED STATES DEPARTMENT OF ENERGY, NOR ANY OF
+# THEIR EMPLOYEES, BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT
+# OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+# STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
+# OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+# *******************************************************************************
+
+require_relative '../spec_helper'
+
+RSpec.describe Ladybug::EnergyModel do
+  it 'has a version number' do
+    expect(Ladybug::EnergyModel::VERSION).not_to be nil
+  end
+
+  it 'has a measures directory' do
+    extension = Ladybug::EnergyModel::Extension.new
+    expect(File.exist?(extension.measures_dir)).to be true
+  end
+
+  it 'has a files directory' do
+    extension = Ladybug::EnergyModel::Extension.new
+    expect(File.exist?(extension.files_dir)).to be true
+  end
+
+  it 'has a valid schema' do
+    extension = Ladybug::EnergyModel::Extension.new
+    expect(extension.schema.nil?).to be false
+    expect(extension.schema_valid?).to be true
+    expect(extension.schema_validation_errors.empty?).to be true
+  end
+
+  it 'can load and validate example schedule ruleset' do 
+    openstudio_model = OpenStudio::Model::Model.new
+    file = File.join(File.dirname(__FILE__), '../files/schedule_ruleset.json')
+    model = Ladybug::EnergyModel::ScheduleRuleset.read_from_disk(file)
+    expect(model.valid?).to be true
+    expect(model.validation_errors.empty?).to be true
+    openstudio_model = model.to_openstudio(openstudio_model)
+    expect(openstudio_model).not_to be nil
+  end
+
+  it 'can load and validate example schedule ruleset 1' do 
+    openstudio_model = OpenStudio::Model::Model.new
+    file = File.join(File.dirname(__FILE__), '../files/schedule_ruleset_1.json')
+    model = Ladybug::EnergyModel::ScheduleRuleset.read_from_disk(file)
+    expect(model.valid?).to be true
+    expect(model.validation_errors.empty?).to be true
+    openstudio_model = model.to_openstudio(openstudio_model)
+    expect(openstudio_model).not_to be nil
+  end
```

Comment:
We should add more assertions that all the schedules are in the openstudio model, they have the right day schedules, with the right values, etc.